### ISSUE TITLE
WebRTC Video/Audio Chat

### ISSUE THREAD

**Issue by capensisma** [sdelements/lets-chat#633]
Hello all :)

Can you please add WebRTC Video/Audio Chat it's very simple since it's already implemented in plenty of webRTC examples .... and It will add an awesome feature to this awesome project :)

**Comment by vwbusguy** [comment]
Is this to do conference calls via lets-chat?  It would probably be more efficient to have one of the many existing audio/video platforms connect to the lets-chat API for authentication or as an intermediary plugin/bot than to incorporate this natively.

**Comment by capensisma** [comment]
@vwbusguy it should be natively added!!

since this is a main feature and related to chatting!

the current feature for chatting is text messages only! why not add audio and video as well? at least for a start between two users? or more? there is plenty and plenty of code ready to use for this!!!

**Comment by vwbusguy** [comment]
It exists for instant messaging style chat, but is non-existent for IRC and chatops.  It would also mean a UI overhaul.  I'm not saying it isn't possible, but rather that I suspect it to be beyond the scope of most users' use cases.

"there is plenty and plenty of code ready to use for this!!!" - All the more reason for a plugin to connect to an already maintained code-base or service rather than re-inventing the wheel.

**Comment by capensisma** [comment]
Ok Scott I got it!

Now how can I add this as a plugin and get it working??

**Comment by chymian** [comment]
may I chime in and suggest jitsi meet and jitsi videobridge. these are already webRTC-XMPP based/connected services.
and yes a videochat 1-to-1 and video-group-conferences are really missing in such a tool.
since I'm not a coder I can't help with the plugin but I could help with install howtos/ansible roles for the videobridge install. let me know.

**Comment by vialcollet** [comment]
I agree that this would be useful to be able to easily continue chat conversation to audio or video.
This is all communication...
